docs/reviews/SINK_CHECKLIST.md

```diff
@@ -0,0 +1,32 @@
+This is a checklist to use whilst reviewing code for Vector's sinks.
+
+## Logic
+
+- [ ] Does it work? Do you understand what it is supposed to be doing?
+- [ ] Does the retry logic make sense?
+- [ ] Are they emitting the correct metrics?
+- [ ] Are the tests testing that they are emitting the correct metrics?
```

Comment:
These two seem like they could be combined. If there are tests, then it is emitting the metrics :)